### PR TITLE
View note page.

### PR DESCRIPTION
**Pull request by rutu14**
image

- The view note page can be accessed by clicking on the Home Page cards.
- The info icon on hover shows the details of the particular note opened.
- The arrow icon on click opens up the action menu for the note.

### CODE REVIEW

**Review by tonymike99 — APPROVED** [review]
Neat code. Very nicely spaced. Good to merge.

File: css/style.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
 @import url(home.css);
```
Comment (iRohitGaur) [inline css/style.css:5]: the UI is really nice

File: pages/view-note.html [hunk 2]
```diff
@@ -25,8 +25,57 @@ <h1 class="heading3 medium brand-name">Doodles</h1>
...
+            </p>
+            <br>
+            <p class="text4 regular">
+                The cartoons are known for some of the most violent cartoon gags ever devised in theatrical animation: Tom may use axes, hammers, firearms, firecrackers, explosives, traps and poison to kill Jerry. On the other hand, Jerry's methods of retaliation are far more violent, with frequent success, including slicing Tom in half, decapitating him, shutting his head or fingers in a window or a door, stuffing Tom's tail in a waffle iron or a mangle, kicking him into a refrigerator, getting him electrocuted, pounding him with a mace, club or mallet, letting a tree or electric pole drive him into the ground, sticking matches into his feet and lighting them, tying him to a firework and setting it off, and so on. While Tom and Jerry has often been criticized as excessively violent, there is no blood or gore in any scene.</p>
+                <br>
```
Comment (iRohitGaur) [inline pages/view-note.html:69]: any reason to use `<br>` here instead of handling it in css?

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: pages/view-note.html (hunk 2 above)
Comment (rutu14) [reply pages/view-note.html:69]: Currently haven't defined CSS for content that is seen as note content. Will define it further.

**Pull request merged**